.catalog-onboard-pipeline.yaml

```diff
@@ -1,49 +1,15 @@
 ---
 apiVersion: v1
 offerings:
-  - name: deploy-arch-ibm-slz-vsi
+  - name: deploy-arch-ibm-slz-vsi 
```

Comment:
the trailing whitespace will fail the pipeline, so please remove it. If you run `pre-commit run --all-files` locally it would fix this

Reply:
`deploy-arch-ibm-slz-vsi` can't be your programatic name. Thats the one for landing zone VSI DA. You seems to have `deploy-arch-ibm-wazi` in your ibm_catalog.json so it needs to match that